pkgs/development/interpreters/clips/default.nix

```diff
@@ -1,14 +1,14 @@
 { lib, stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
-  version = "6.4.1";
+  version = "6.40";
```

Comment:
This is actually a downgrade. Version 6.4.1 was released in 2023-04. Version 6.40 was released in 2021-06.
https://sourceforge.net/projects/clipsrules/files/CLIPS/
I have reported that to repology, but I'm not sure if that will (eventually) solve the problem for nixpkgs auto-update bot.